### ISSUE TITLE
Request: Visual Feedback When Adding New Series in UI

### ISSUE THREAD

**Issue by jakem742** [mylar3/mylar3#1210]
**Describe the request**
In Mylar series search, change the series result row colour (to blue?) when selecting "Add this Comic". Blue seems to be the colour for "grabbed but not processed" things in Mylar, so it makes sense to me?

**How did you come up with the idea?**
When searching for new series in Mylar, there is no feedback to indicate which series you have selected. The only way to identify which series you have already selected is by re-running the search.

**Additional information**
This probably isn't an issue if you're only after one specific series, but it makes it difficult to keep track of things when you're selecting 5+ series in the results and can't remember which you've clicked.

**Comment by evilhero** [comment]
Blue = snatched. Depending on the provider it can mean different things. Mainly that ddl its being queued to download, whereas with nzb/torrents it has been sent to the download client to download.

It will flip the status to Snatched after a successful search and send. It will update the status in real time if you're on the series screen for the given issue. If you're using the back browser button to go back and forth, you're using cached values and probably viewing stale data.

I'm still trying to figure out what it is that you're actually describing - as you mention add this comic, which is adding a series to the warchlist, and then talking about issues downloading, which are completely different methods.

However, I have a feeling you're talking about the search results page that displays when you search for a series and then try to add it and it doesn't update the status for the newly added series (so that it's displaying as in your watchlist, but on the results page) . If so, this is already a known issue - the solution is to just refresh the page until such time as a fix can be done.

**Comment by jakem742** [comment]
Yeah sorry, I'm talking about adding a new series to Mylar (using the search bar in the top left corner), not managing issues within a series. I am assuming this is what you're referring to as adding a series to the watchlist. 

I didn't realise you could refresh the search results page without losing the current search results. That works. :)
